### PR TITLE
Adding optional parameter to specify cross origin requests

### PR DESCRIPTION
**Pull request by travislbrundage**

### CODE REVIEW

File: src/services/MapConfigTransformService.js [hunk 1]
```diff
@@ -183,7 +183,11 @@ class MapConfigTransformService {
       thumbnail: thumbnail
     };
   }
-  transform(data, opt_errors, opt_tileServices) {
+  transform(data, opt_errors, opt_tileServices, opt_crossOrigin) {
```
Comment (bartvde) [inline src/services/MapConfigTransformService.js]: I'd either pass in a string (so ```use-credentials``` as the value), or rename this to opt_useCredentials

File: src/services/MapConfigTransformService.js [hunk 2]
```diff
@@ -235,15 +239,15 @@ class MapConfigTransformService {
         layerConfig.source = {
           type: 'OSM',
           properties: {
-            crossOrigin: 'anonymous'
+            crossOrigin: crossOrigin
```
Comment (bartvde) [inline src/services/MapConfigTransformService.js]: I would say for OSM sources this will never be needed? Since their tiles will remain freely accessible?

**Comment by travislbrundage** [comment]
I updated the changes as you requested, thank you for reviewing!

**Comment by travislbrundage** [comment]
One moment, small mistake is happening - fixing

**Pull request merged**